packages/xstate-vue/src/xstate-machine.ts

```diff
@@ -0,0 +1,95 @@
+import { interpret, State } from 'xstate';
+import Vue, { VNode } from 'vue';
+
+export default Vue.extend({
+  render(h): VNode {
+    const { current, service, send } = this;
+    return h('div', [
```

Comment:
Probably a naïve question on my part, but is `'div'` necessary? Does it need to be specified, and what if the developer wants a different component for this? Or no component at all? Just thinking out loud 💭

Reply:
it is requeried since vue template must have 1 parent element (it was like this in react before Fragment was introduced)

Reply:
I think this may not be needed. You should be able to do this:
```js
render: VNode {
	const { current, service, send } = this;
    return this.$scopedSlots.default({
        current,
        service,
        send
      })
	},
}
```

I'm going to open a PR to fix this. :)

Reply:
@studnitz while doing this, could you also add some simple tests for this?

Reply:
Yea, was planning on that anyways.

Reply:
@studnitz do you know when you will have time to work on this? I dont want to rush you of course, just would love to know approximate timing.

Reply:
I'm trying to tackle it this evening!